src/module/plugins/entry.ts

```diff
@@ -1,6 +1,6 @@
 import { readFileSync } from 'node:fs'
 import { dirname, extname, join } from 'pathe'
-import { createUnplugin } from 'unplugin'
+import type { Plugin } from 'vite'
```

Comment:
unplugin also has hook filter support, so you should be able to stick with it (if wanted)